modern/src/common/components/StatusCard.js

```diff
@@ -57,6 +57,8 @@ const useStyles = makeStyles((theme) => ({
   content: {
     paddingTop: theme.spacing(1),
     paddingBottom: theme.spacing(1),
+    'max-height': '40vh',
+    overflow: 'hidden',
```

Comment:
Why are we hiding overflow?

Reply:
because it overlaps with the actions

Reply:
without overflow: hidden

![localhost_3000_(iPhone SE) (3)](https://github.com/traccar/traccar-web/assets/31902485/7e5bb0a5-02ac-4a55-8786-ef27212c712e)

Reply:
Screenshot please. And also wouldn't it mean that it's not the content is not accessible?

Reply:
yes, the content overflow content will not accesible

Reply:
That's not an acceptable solution then. It should become scrollable if you want to limit the height.

Reply:
Changed to overflow: scroll
![localhost_3000_(iPhone SE) (4)](https://github.com/traccar/traccar-web/assets/31902485/14d711e8-75f6-4efc-af51-9d3b0c4477a8)